boards/kernel_layout.ld

```diff
@@ -85,6 +85,7 @@ SECTIONS
          * shouldn't have any effect.
          */
         KEEP(*(.riscv.start));
+        . = ALIGN(64);
         KEEP(*(.riscv.trap));
```

Comment:
The trap handler must be aligned to a multiple of 64 bytes. With `global_asm!()` this could be inserted in the assembly for the start trap function. However, using functions with `asm!()` I think the only place to add this is here. Does anyone know a better way?

Reply:
To remove the padding on boards without this, this works:

            . = DEFINED(_start_trap) ? ALIGN(64) : ALIGN(1);

Though whether it's worth the worst-case 63 bytes, I'm not sure I care as much